src/site-server.ts

```diff
@@ -168,7 +168,15 @@ export class SiteServer {
 
 	async executeWpCliCommand(
 		args: string,
-		{ targetPhpVersion }: { targetPhpVersion?: string } = {}
+		{
+			targetPhpVersion,
+			includePlugins,
+			includeThemes,
```

Comment:
```suggestion
			includePlugins = false,
			includeThemes = false,
```
Nit, but I think this clarifies intent